snowplow/iglu-client-embedded/schemas/com.metabase/structured_action/jsonschema/1-0-0

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Generic event to track interactions and actions that happen within Metabase. This should be used as a default schema for tracking events. If more specific context is needed, we can use custom context entities (https://docs.snowplow.io/docs/understanding-your-pipeline/entities/).",
+    "self": {
+      "vendor": "com.metabase",
+      "name": "structured_action",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "name": {
+        "description": "Name of the action. Noun (target) + Verb in the past (action) which define the action taken - e.g. question-created, dashboard-updated, dashboard-auto-apply-filter-enabled",
+        "type": "string",
+        "maxLength": 1024
+      },
+      "target_id": {
+        "description": "(Optional) ID of the entity that the action was performed on. E.g. the ID of the question that was created in a question_created event.",
+        "type": [
+            "string",
+            "null"
+        ],
+        "maxLength": 1024
+      },
+      "triggered_from": {
+        "description": "(Optional) From where the action was taken/triggered. This can be generic like 'dashboard' or also more specific like 'dashboard_top_nav'.",
+        "type": [
+            "string",
+            "null"
+        ],
+        "maxLength": 1024
+      },
+      "duration_ms": {
```

Comment:
Does we need it here in a generic event?

Reply:
Yes I would vote for keeping it. While discussing the schema and looking at old events, we had some cases where we said this can make sense to have here. If you strongly disagree, we can also remove it for now and potentially create a minor schema update in the future if needed